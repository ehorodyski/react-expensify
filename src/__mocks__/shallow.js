import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

export default (component) => {
  return toJSON(shallow(component));
};