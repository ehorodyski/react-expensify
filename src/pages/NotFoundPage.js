import React from 'react';
import { Link } from 'react-router-dom';

export const NotFoundPage = () => (
  <div>
    YOU HIT A HIDDEN PAGE! - <Link to="/">Go home</Link>
  </div>
);