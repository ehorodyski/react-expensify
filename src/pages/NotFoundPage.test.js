import React from 'react';
import shallow from '../__mocks__/shallow';
import { NotFoundPage } from './NotFoundPage';

test('should render NotFoundPage correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});