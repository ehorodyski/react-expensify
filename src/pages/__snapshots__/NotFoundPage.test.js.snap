// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div>
  YOU HIT A HIDDEN PAGE! - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
